### PR TITLE
Bug/timer sync mobile

### PR DESCRIPTION
**Pull request by desperado1802**
#1494
Synced the timers when it's started from Web side, the opposite was working already. Also fixed API calls. Timer status wasn't fetching constantly to check if timer is running, now does every 5 sec

We start the Web App timer:
https://github.com/ever-co/ever-teams/assets/124465103/185b5d4f-7fbb-42b3-80b2-6794d20d7374

Mobile App syncs(starts too):
https://github.com/ever-co/ever-teams/assets/124465103/947f1a4f-df4a-4b1d-b489-19257e16fb83

### CODE REVIEW

File: apps/mobile/app/services/client/queries/timer/timer.ts [hunk 1]
```diff
@@ -4,34 +4,36 @@ import { ITimerTimeslotParams } from "../../../interfaces/ITimer"
...
-	const { data } = await getTimerStatusRequest(
-		{ source: "MOBILE", tenantId, organizationId },
-		authToken,
-	)
+	if (isTimerRunning) {
+		await syncTimeSlotRequest(
+			{
+				tenantId,
+				organizationId,
+				source: "MOBILE",
```
Comment (badalkhatri0924) [inline apps/mobile/app/services/client/queries/timer/timer.ts:15]: @desperado1802 I think this is wrong. 

We have hard coded MOBILE source sync, we have to make it dynamic.

For Example:
If user started timer in WEB or DESKTOP or at other source we have to pass that value in source (If we don't find that value then only we have to pass MOBILE).


Take Reference from:
https://github.com/ever-co/ever-teams/blob/develop/apps/web/app/hooks/features/useTimer.ts#L238


So basically we have to at which source current timer is running, and we have to pass that value.

**Review by badalkhatri0924 — CHANGES_REQUESTED** [review]

**Review by badalkhatri0924 — APPROVED** [review]
